qiskit/aqua/operators/evolutions/evolved_op.py

```diff
@@ -88,9 +89,19 @@ def tensor(self, other: OperatorBase) -> OperatorBase:
 
         return TensoredOp([self, other])
 
-    def compose(self, other: OperatorBase) -> OperatorBase:
-        other = self._check_zero_for_composition_and_expand(other)
+    def expand_with_identity(self, num_qubits: int) -> 'OperatorBase':
```

Comment:
Could we make these private until we find a user-facing use-case? 
```suggestion
    def _expand_with_identity(self, num_qubits: int) -> 'OperatorBase':
```

Reply:
Sure 👍